apps/ui/src/components/checkbox/__tests__/checkbox.test.tsx

```diff
@@ -0,0 +1,67 @@
+import { render, screen, fireEvent } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import Checkbox from '../checkbox';
+
+describe('Checkbox/checkbox', () => {
+	describe('layout', () => {
+		test('should render by default', () => {
+			render(<Checkbox checked={false} onChange={() => {/* No-op */}} />);
```

Comment:
I recommend removing `checked` property so that, the properties will be set as defaults